app/src/org/odk/collect/android/views/ResizingImageView.java

```diff
@@ -157,41 +158,22 @@ public void setFullScreen(){
         }
     }
 
-    public Pair<Integer,Integer> getWidthHeight(int widthMeasureSpec, int heightMeasureSpec, Drawable drawable, double scaleFactor){
+    public Pair<Integer,Integer> getWidthHeight(int widthMeasureSpec, int heightMeasureSpec, double scaleFactor){
```

Comment:
The method parameter 'scaleFactor' overshadows the class-level 'scaleFactor' field. This is confusing.